test/cog/repository/bundles_test.exs

```diff
@@ -43,7 +43,7 @@ defmodule Cog.Repository.BundlesTest do
     assert %BundleVersion{version: ^fixed_version, bundle: %Bundle{name: "site"}} = Bundles.site_bundle_version
   end
 
-  Enum.each(["operable", "site", "user"], fn(protected_name) ->
+  Enum.each(["operable", "user"], fn(protected_name) ->
```

Comment:
We should still test that somebody can't come in and create a bundle named "site".